build.gradle

```diff
@@ -200,20 +207,21 @@ publishing {
     }
 }
 
-String getProjectProperty(String propertyName)
-{
+String getProjectProperty(String propertyName) {
     String property = ""
-    if (hasProperty(propertyName))
-    {
+    if (hasProperty(propertyName)) {
         property = this.properties[propertyName]
     }
     return property
 }
 
-wrapper.gradleVersion = '4.6'
+wrapper {
```

Comment:
Can we surround this with a check if the wrapper tasks exists?
```java
if (tasks.findByName('wrapper')) {
    wrapper {
        gradleVersion = '4.9'
    }
}
```
The reason for this is that the wrapper tasks only exists on root projects in gradle. If you include JDA as a subproject (I have it as subproject for my test bot for example) even loading the project already fails with 
```Could not get unknown property 'wrapper' for project ':JDA' of type org.gradle.api.Project.```